.gitignore

```diff
@@ -1,2 +1,3 @@
 *.orig
 build
+.idea/
```

Comment:
This could be removed, if you need to ignore IDE-specific files you can add them to your global .gitignore https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer